extensions/qotp/src/menu-bar-qotp.tsx

```diff
@@ -0,0 +1,28 @@
+import { MenuBarExtra } from "@raycast/api";
+import { ExtraList } from "./data";
+
+// package.json command
+// {
+//   "name": "menu-bar-qotp",
+//   "title": "QOTP",
+//   "description": "QOTP is a simple tool to generate OTP code",
+//   "mode": "menu-bar",
+//   "interval": "1s"
+// }
+
+export default function Command() {
+  return (
+    <MenuBarExtra icon="otp.png" tooltip="点击查看OTP">
```

Comment:
At the moment, Raycast doesn't support localization and only supports US English. Therefore, please avoid introducing your custom way to localize your extension.